### PR TITLE
Create election page

### PR DESCRIPTION
**Pull request by meyeralex31**
https://user-images.githubusercontent.com/14840649/219905031-13460647-ad1c-4b49-aad4-f641fe0f62f9.mp4

I also create the normal user registration page as well. 
React App - Google Chrome 2_18_2023 7_08_26 PM

### CODE REVIEW

File: src/RegisterElectionPage/Collectors.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import List from "@mui/material/List";
+import ListSubheader from "@mui/material/ListSubheader";
+import ListItem from "@mui/material/ListItem";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+
+const Collectors = () => {
+  const collectors = [
```
Comment (eashwarmp) [inline src/RegisterElectionPage/Collectors.jsx:9]: Is there any reason why the collectors are repeating? Is this like a temporary placeholder for us to add more in the future or something?

**Review by eashwarmp — CHANGES_REQUESTED** [review]

File: src/RegisterElectionPage/Collectors.jsx (hunk 1 above)
Comment (meyeralex31) [reply src/RegisterElectionPage/Collectors.jsx:9]: Ya this was just a place holder. I was just doing it to make sure the sizing of the scrolling box was correct

Comment (meyeralex31) [reply src/RegisterElectionPage/Collectors.jsx:9]: None of this is going to work right and still needs connected to the backend. My goal right now is to finish the UI

Comment (eashwarmp) [reply src/RegisterElectionPage/Collectors.jsx:9]: Gotcha sounds good!

**Review by eashwarmp — APPROVED** [review]
PR Looks good to me

**Pull request merged**